customTests/azure_ib_write_bw_gdr.nhc

```diff
@@ -86,7 +86,7 @@ function check_ib_bw_gdr(){
       pid_array+=($!)
       (( dev_idx++ ))
    done
-   wait  ${pid_array[@]}
+   wait  "${pid_array[@]}"
```

Comment:
Why did we change this?

Reply:
I saw an error from ShellCheck about placing it inside quotes